config/controllerrole.yaml

```diff
@@ -9,6 +9,10 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: kpack-controller-admin
+aggregationRule:
```

Comment:
split `rules` and `aggreationRule` to use different roles. The aggregation will overwrite the rules rather then augment them.